### DISCUSSION TITLE
after nuxt3 is packaged, can’t see the dom content when viewing the source code, only js.

### DISCUSSION THREAD

**Discussion by JIAFENG123** [nuxt/nuxt#26067] [Questions]
大佬们好，我使用nuxt3写的一个项目，npm run dev执行之后，本地查看右键可以查看源代码dom结构，但是build之后，生产环境的打开就是空的，看不到dom，求救~

Hello everyone, I wrote a project using nuxt3. After executing npm run dev, I can view the source code dom structure by right-clicking locally. However, after building, the production environment is empty when I open it, and I can’t see the dom. Please help~

![image](https://github.com/nuxt/nuxt/assets/34330528/af122fc7-d6f9-4eb3-8540-c964eb94c01f)

**Comment by xjccc** [comment]
Would you be able to provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs#create-a-minimal-reproduction)

**Reply by JIAFENG123** [reply]
Hello, I don’t know where the problem is. It’s hard to give an example. I’m using nuxt3 + arcodesign. It works locally but not in production.

**Reply by xjccc** [reply]
emm.. I think you can provide a mini-example with your config.

**Reply by JIAFENG123** [reply]
ok, thanks

**Reply by JIAFENG123** [reply]
emmm..I seem to have found the problem. I changed the arcodesign components used in my layouts to basic tags like main、div, and it seems to have taken effect. emmm...   thanks!
![image](https://github.com/nuxt/nuxt/assets/34330528/062f164e-0628-440f-ad29-3baad7e46fae)

**Reply by JIAFENG123** [reply]
But I don’t understand why the previous local can，，emmm

**Reply by xjccc** [reply]
I don't know this. I never use arcodesign before.
If you can't see the structure from source. There are some things that may have been overlooked.

1. `ClientOnly` in your components
2. `.client.vue` at your file
3. `ssr: false` in your config

or some thing in `arcodesign` use `onMounted` to render in client
